faunadb-scala/src/main/scala/faunadb/values/Value.scala

```diff
@@ -102,6 +102,8 @@ object Value {
 
   /** Create a date value. */
   def apply(localdate: LocalDate): Value = DateV(localdate)
+
+  implicit def from[T](obj: T)(implicit encoder: Encoder[T]): Value = encoder.encode(obj)
```

Comment:
I don't see other good place for this method considering the way Scala search for its implicit implementations.

So, if we're leaving it here, I would suggest to replace the other `apply` methods with one that does the same as `from`. Just to keep it clean.